doc/subversion_and_git_integration.md

```diff
@@ -94,23 +94,32 @@ PerlLoadModule Apache::OpenProjectAuthentication
 &lt;/VirtualHost&gt;
 </pre>
 
-## Automatically create repositories with reposman.rb
+## Automatically create repositories
 
-The reposman.rb script can create repositories for your newly created OpenProject projects.
-It is useful when run from a cron job (so that repositories appear 'magically' some time after you created
-a project in the OpenProject administration view).
+You can create repositories explicitly on the filesystem using managed repositories.
+Enable managed repositories for each SCM vendor individually using the templates
+defined in configuration.yml.
 
-<pre>
-ruby extra/svn/reposman.rb \
-  --openproject-host "http://127.0.0.1:3000" \
-  --owner "www-data" \
-  --group "openproject" \
-  --public-mode '2750' \
-  --private-mode '2750' \
-  --svn-dir "/srv/openproject/svn" \
-  --url "file:///srv/openproject/svn" \
-  --key "REPLACE WITH REPOSITORY API KEY" \
-  --scm Subversion \
-  --verbose
-</pre>
+### reposman.rb
+
+This functionality was previously provided in an asynchronous manner using reposman.rb.
+This script has been integrated into OpenProject.
+Please remove any existing cronjobs that still use this script.
+
+To convert repositories previously created by reposman.rb into managed repositories,
+use the following command:
+
+    $ bundle exec rake scm:migrate:reposman[URL prefix (, URL prefix, ...)]
```

Comment:
I don't think this should be limited to repos having been created via reposman. In a scenario, where a repository is moved to openproject for one reason or the other, this functionality is necessary as well.